Tribler/Core/Modules/MetadataStore/OrmBindings/metadata.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+
+from pony import orm
+
+from Tribler.Core.Modules.MetadataStore.serialization import MetadataTypes, MetadataPayload, DeletedMetadataPayload
+from Tribler.pyipv8.ipv8.keyvault.crypto import ECCrypto
+
+
+
+def define_binding(db):
+    class Metadata(db.Entity):
+        rowid = orm.PrimaryKey(int, auto=True)
+        metadata_type = orm.Discriminator(int)
+        _discriminator_ = MetadataTypes.TYPELESS.value
+        # Want to make signature unique=True for safety, but can't do it in Python2 because of Pony bug #390
+        signature = orm.Optional(buffer)
+        timestamp = orm.Optional(datetime, default=datetime.utcnow)
+        tc_pointer = orm.Optional(int, size=64, default=0)
+        public_key = orm.Optional(buffer, default='\x00' * 74)
+        addition_timestamp = orm.Optional(datetime, default=datetime.utcnow)
+        deleted = orm.Optional(bool, default=False)
+        _payload_class = MetadataPayload
+        _my_key = None
+
+        def __init__(self, *args, **kwargs):
+            super(Metadata, self).__init__(*args, **kwargs)
+            # If no key/signature given, sign with our own key.
+            if "public_key" not in kwargs or (kwargs["public_key"]==self._my_key and "signature" not in kwargs):
+                self.sign(self._my_key)
+
+        def _serialized(self, key=None):
```

Comment:
Please add some comments here, especially that it returns the serialized data and a signature as a tuple.

Reply:
done